### PR TITLE
support unstable jenkins status

### PR DESCRIPTION
**Pull request by grosser**
@rdhanoa @sandlerr 

https://zendesk.airbrake.io/projects/95346/groups/1786680229285535838/notices/1839942896368086033?tab=notice-list

### CODE REVIEW

**Comment by grosser** [comment]
also fixing xss ... and making it not blow up on every new status

File: plugins/jenkins/app/helpers/jenkins_helper.rb [hunk 1]
```diff
@@ -29,17 +31,16 @@ def jenkins_status_panel(deploy, jenkins_job)
...
       status = 'warning'
       status_message = jenkins_job_status
-    else
-      status = mapping.fetch(jenkins_job_status)
-      status_message = result.fetch(jenkins_job_status)
     end
 
     content = "Jenkins build #{jenkins_job.name} for #{deploy.stage.name} #{status_message}"
-    content_tag :a, href: jenkins_job_url, target: "_blank" do
-      content_tag :div, content.html_safe, class: "alert alert-#{status}"
```
Comment (sandlerr) [inline plugins/jenkins/app/helpers/jenkins_helper.rb]: @rdhanoa does this need to include html that actually gets parsed or can it be escaped?

Comment (grosser) [inline plugins/jenkins/app/helpers/jenkins_helper.rb]: nope :D

Comment (sandlerr) [inline plugins/jenkins/app/helpers/jenkins_helper.rb]: :+1: :shipit:

**Pull request merged**